sdk/tools.go

```diff
@@ -0,0 +1,9 @@
+//go:build tools
```

Comment:
As explained in the PR description, this file will be ignored by the standard go build process as it's using a custom tag.